UI/GovernmentWnd.cpp

```diff
@@ -1519,9 +1560,20 @@ void GovernmentWnd::CompleteConstruction() {
     m_policy_palette->ClearPolicySignal.connect(
         boost::bind(&GovernmentWnd::MainPanel::ClearPolicy, m_main_panel, _1));
 
+    m_policy_size_buttons = GG::Wnd::Create<GG::RadioButtonGroup>(GG::Orientation::HORIZONTAL);
+    m_policy_size_buttons->AddButton(
+        GG::Wnd::Create<CUIStateButton>("100%", GG::FORMAT_LEFT, std::make_shared<CUIRadioRepresenter>()));
```

Comment:
these % are somewhat meaningless and arbitrary... just some icon for big, medium, and small would be more appropriate